airflow/config_templates/config.yml

```
@@ -335,6 +335,15 @@
       type: string
       example: ~
       default: "True"
+    - name: store_dag_code
+      description: |
+        Whether to persist DAG files code in DB.
+        If set to True, Webserver reads file contents from DB instead of
+        trying to access files in a DAG folder.
+      version_added: 2.0.0
+      type: string
+      example: ~
+      default: "False"
```

To address Kaxil's comment I think we can do this

```suggestion
        Whether to persist DAG files code in DB.
        If set to True, Webserver reads file contents from DB instead of
        trying to access files in a DAG folder. Defaults to same as the
        store_serialized_dags setting
      version_added: 2.0.0
      type: string
      example: ~
      default: "%(store_serialized_dags)s"
```

This will use the "Basic interpolation" built in to config parser https://docs.python.org/3/library/configparser.html#configparser.BasicInterpolation

Example:

```ini
[Paths]
home_dir: /Users
my_dir: %(home_dir)s/lumberjack
my_pictures: %(my_dir)s/Pictures
```

---

Thanks. I was too quick with committing your first suggestion so that I could not accept your first suggestion anymore. But I have manually added it.